backend/apps/moderation/audit_utils.py

```
@@ -0,0 +1,16 @@
+from typing import Optional
+
+
+def safe_parse_date(date_str: Optional[str]):
+    if not date_str:
+        return None
+    try:
+        # Accept YYYY-MM-DD
+        from datetime import datetime
+
+        return datetime.strptime(date_str, "%Y-%m-%d").date()
+    except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)